invenio_app_rdm/records_ui/views/filters.py

```diff
@@ -190,9 +190,12 @@ def custom_fields_search(field, field_value, field_cfg=None):
         namespace_string = r"\:".join(namespace_array) + f".{localised_title}.{locale}"
     else:
         namespace_string = r"\:".join(namespace_array)
+        if (field_cfg or {}).get("link_by_id", False):
```

Comment:
why not something like

```suggestion
        if (field_cfg or {}).get("landing_page_search_attr", False):
            namespace_string += f".{landing_page_search_attr}"
```

naming can be changed ofc....I am thinking more on allowing users to configure the attribute they would like to search on. The default can remain as it was with either the localized title or just the raw value. I am afraid that is too rigid to have a property only for id. @kpsherva @ntarocco wdyt?

Reply:
I agree, more flexible